testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/mock/JobWorkerMockBuilderImpl.java

```diff
@@ -51,14 +62,34 @@ public JobWorkerMock thenComplete(final Map<String, Object> variables) {
     return withHandler(
         (jobClient, job) -> {
           LOGGER.debug(
-              "Mock: Complete job with variables {} [job-type: '{}', job-key: '{}']",
-              variables,
-              jobType,
-              job.getKey());
+              "{} with variables {}", logMessagePrefix.apply(ACTION_COMPLETE_JOB, job), variables);
+
           jobClient.newCompleteCommand(job).variables(variables).send().join();
         });
   }
 
+  @Override
+  public JobWorkerMock thenCompleteWithExampleData() {
+    return withHandler(
+        (jobClient, job) -> {
+          final String logMessagePrefix = this.logMessagePrefix.apply(ACTION_COMPLETE_JOB, job);
+
+          try {
+            final String exampleDataVariables =
+                exampleDataReader.readExampleData(
+                    job.getProcessDefinitionKey(), job.getElementId());
+
+            LOGGER.debug("{} with example data {}", logMessagePrefix, exampleDataVariables);
+            jobClient.newCompleteCommand(job).variables(exampleDataVariables).send().join();
+          } catch (final BpmnExampleDataReaderException e) {
+
+            LOGGER.warn(
+                "{} without example data due to errors. {}", logMessagePrefix, e.getMessage(), e);
```

Comment:
🔧 (same as in the Context) Printing the error duplicated the failure message. We should remove the exception from the log message.


```suggestion
                "{} without example data due to errors. {}", logMessagePrefix, e.getMessage());
```